authors/apps/articles/tests/test_data.py

```diff
@@ -0,0 +1,23 @@
+
+valid_article = {
+    "article": {
+        "title": "When ooh in oo updated  ocean",
+        "description": "this is the description,.",
+        "body": "this is the body"
+    }
+}
+updated_article = {
+    "article": {
+        "title": "When ooh in oo updated  ocean",
+        "description": "this is the description,.",
+        "body": "this is the body"
+    }
+}
+valid_tagged_article = {
+    "article": {
+        "title": "When ooh in oo updated  ocean",
+        "description": "this is the description,.",
+        "body": "this is the body",
+        "tag_list": ["testtag", "test3", "test4"]
```

Comment:
I believe that tagging is another story all together you may consider revising your work to handle only Article Crud

Reply:
@wasibani-roy thank you, I have worked on that.